src/sensitivities/array.jl

```diff
@@ -1,3 +1,9 @@
 # Let the user get the `size` and `length` of `Node`s.
 Base.size(x::Node, dims...) = size(x.val, dims...)
 Base.length(x::Node) = length(x.val)
+
+# Sensitivity for the first argument of `reshape`.
+import Base.reshape
+@explicit_intercepts reshape Tuple{∇RealArray, Vararg{Int64}} [true, false]
```

Comment:
Just took a look at the method table for `reshape` and it has a method which accepts a variable sized `Tuple` as the second argument, so you can do stuff of the form `reshape(randn(5, 5), (25,))` as well as `reshape(randn(5, 5), 25)`. I think you might need another use if `@explicit_intercepts` to handle that case.

Reply:
Good catch. I'll take care of that shortly.